content/en/docs/concepts/security/service-accounts.md

```diff
@@ -196,6 +196,31 @@ or using a custom mechanism such as an [authentication webhook](/docs/reference/
 You can also use TokenRequest to obtain short-lived tokens for your external application.
 {{< /note >}}
 
+### Enforcing Mountable Secrets using Annotations {#enforce-mountable-secrets}
+
+Kubernetes provides an annotation called `kubernetes.io/enforce-mountable-secrets` that you can add to your ServiceAccounts.
+When this annotation is applied, the service account's secrets can only be mounted on specified types of resources, enhancing the security posture of your cluster.
+
+#### How to Add the Annotation
+
+You can add the annotation to a ServiceAccount using a manifest:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    kubernetes.io/enforce-mountable-secrets: "true"
+  name: my-serviceaccount
+  namespace: my-namespace
+```
+
+#### Effect on the Cluster
+
+When this annotation is set to true, the Kubernetes control plane ensures that the secrets belonging to this ServiceAccount are only mountable on the resource types specified in the annotation value.
+For more information about the authentication process, refer to
+[kubernetes.io/enforce-mountable-secrets {#enforce-mountable-secrets}](/docs/reference/labels-annotations-taints/_index.md#kubernetesioenforce-mountable-secrets-enforce-mountable-secrets).
```

Comment:
```suggestion
[the documentation on this annotation](/docs/reference/labels-annotations-taints/#enforce-mountable-secrets).
```

Reply:
I prefer the text @chansuke proposed, here. This link is to a reference page @tengqm and not to documentation about using the annotation.